content/en/community.md

```diff
@@ -3,28 +3,77 @@ title: Community
 sidebar: false
 ---
 
-NumPy offers a number of communication channels available to individuals interested in using and contributing to the project.
 
-We welcome contributions and contributors of all kinds - whether they come as contributions to code, participation in the community, opening issues and pointing out bugs, or simply sharing your work with others.
+# Community
```

Comment:
Note: this isn't needed, it's already given above in the `title:` metadata. Also, `<h1>` section headers don't work, start from h2 (`##`).